🌐 [i18n-KO] Translated `vision-encoder-decoder.md` to Korean (#39563)
huggingface/transformers: docs/source/ko/model_doc/vision-encoder-decoder.md

Zhoujun Li, Furu Wei의 [TrOCR: Transformer-based Optical Character Recognition with Pre-trained Models](https://huggingface.co/papers/2109.10282)에서 입증되었습니다.

이러한 [`VisionEncoderDecoderModel`]이 훈련/미세 조정된 후에는 다른 모델들과 마찬가지로 저장/가져올 수 있습니다 (자세한 내용은 아래 예제를 참조하세요).

응용 예시로는 이미지 캡셔닝이 있습니다. 여기서는 인코더가 이미지를 인코딩하고, 이후 자기회귀 언어 모델이 캡션을 생성합니다. 다른 예시로는 광학 문자 인식이 있습니다. [`VisionEncoderDecoderModel`]의 인스턴스인 [TrOCR](trocr)을 참조하세요.

## 모델 설정에서 `VisionEncoderDecoderModel` 랜덤 초기화[[randomly-initializing-visionencoderdecodermodel-from-model-configurations]]

[`VisionEncoderDecoderModel`]은 인코더와 디코더 설정에서 랜덤하게 초기화할 수 있습니다. 다음 예제에서는 인코더에 기본 [`ViTModel`] 설정을,
디코더에 기본 [`BertForCausalLM`] 설정을 사용하여 이를 수행하는 방법을 보여줍니다.

```python
>>> from transformers import BertConfig, ViTConfig, VisionEncoderDecoderConfig, VisionEncoderDecoderModel

>>> config_encoder = ViTConfig()
>>> config_decoder = BertConfig()

>>> config = VisionEncoderDecoderConfig.from_encoder_decoder_configs(config_encoder, config_decoder)
>>> model = VisionEncoderDecoderModel(config=config)
```

## 사전훈련된 인코더와 디코더에서 `VisionEncoderDecoderModel` 초기화[[initialising-visionencoderdecodermodel-from-a-pretrained-encoder-and-a-pretrained-decoder]]

> [`VisionEncoderDecoderModel`]은 사전훈련된 인코더 체크포인트와 사전훈련된 디코더 체크포인트에서 초기화할 수 있습니다. 사전훈련된 Transformer 기반 비전 모델(*예:* [Swin](swin))은 인코더 역할을 할 수 있으며, 사전훈련된 오토인코딩 모델(*예:* BERT), 사전훈련된 인과적 언어 모델링(causal language modeling)(*예:* GPT2), 그리고 시퀀스-투-시퀀스 모델의 사전훈련된 디코더 부분(*예:* BART의 디코더)은 모두 디코더로 사용할 수 있습니다.

Reviewer: ```suggestion
[`VisionEncoderDecoderModel`]은 사전 훈련된 인코더 체크포인트와 사전 훈련된 디코더 체크포인트에서 초기화할 수 있습니다. 사전 훈련된 Transformer 기반 비전 모델(*예:* [Swin](swin))은 인코더 역할을 할 수 있으며, 사전 훈련된 오토인코딩 모델(*예:* BERT), 사전 훈련된 인과적 언어 모델링(causal language modeling)(*예:* GPT2), 그리고 시퀀스-투-시퀀스 모델의 사전 훈련된 디코더 부분(*예:* BART의 디코더)은 모두 디코더로 사용할 수 있습니다.
```

> [`VisionEncoderDecoderModel`]은 사전훈련된 인코더 체크포인트와 사전훈련된 디코더 체크포인트에서 초기화할 수 있습니다. 사전훈련된 Transformer 기반 비전 모델(*예:* [Swin](swin))은 인코더 역할을 할 수 있으며, 사전훈련된 오토 인코딩 모델(*예:* BERT), 사전훈련된 인과적 언어 모델(*예:* GPT2), 그리고 시퀀스-투-시퀀스 모델의 사전훈련된 디코더 부분(*예:* BART의 디코더)은 모두 디코더로 사용할 수 있습니다.

Author: ```suggestion
[`VisionEncoderDecoderModel`]은 사전훈련된 인코더 체크포인트와 사전 훈련된 디코더 체크포인트에서 초기화할 수 있습니다. 사전 훈련된 Transformer 기반 비전 모델(*예:* [Swin](swin))은 인코더 역할을 할 수 있으며, 사전 훈련된 오토 인코딩 모델(*예:* BERT), 사전 훈련된 인과적 언어 모델(*예:* GPT2), 그리고 시퀀스-투-시퀀스 모델의 사전 훈련된 디코더 부분(*예:* BART의 디코더)은 모두 디코더로 사용할 수 있습니다.
```

디코더로 선택하는 아키텍처에 따라 교차 어텐션(cross-attention) 레이어가 랜덤하게 초기화될 수 있습니다.
사전훈련된 인코더와 디코더 체크포인트에서 [`VisionEncoderDecoderModel`]을 초기화하려면 [*Warm-starting-encoder-decoder blog post*](https://huggingface.co/blog/warm-starting-encoder-decoder)에서 보여준 바와 같이 다운스트림 작업에서 모델을 미세 조정해야 합니다.
이를 위해 `VisionEncoderDecoderModel` 클래스는 [`VisionEncoderDecoderModel.from_encoder_decoder_pretrained`] 메소드를 제공합니다.

```python
>>> from transformers import VisionEncoderDecoderModel

>>> model = VisionEncoderDecoderModel.from_encoder_decoder_pretrained(
...     "microsoft/swin-base-patch4-window7-224-in22k", "google-bert/bert-base-uncased"
... )
```

> ## 기존 `VisionEncoderDecoderModel` 체크포인트 가져오기 및 추론 수행[[loading-an-existing-visionencoderdecodermodel-checkpoint-and-perform-inference]]

Reviewer: ```suggestion
## 기존 `VisionEncoderDecoderModel` 체크포인트 불러와 추론하기[[loading-an-existing-visionencoderdecodermodel-checkpoint-and-perform-inference]]
```
